torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -893,17 +893,6 @@ AOTITorchError aoti_torch_proxy_executor_call_function(
   });
 }
 
-void aoti_torch_check(
-    bool cond,
-    const char* func,
-    const char* file,
-    uint32_t line,
-    const char* msg) {
-  if (C10_UNLIKELY_OR_CONST(!cond)) {
```

Comment:
Is `C10_UNLIKELY_OR_CONST` causing CPU benchmark perf regression? We could simply remove it here.

Reply:
No. CPU benchmark perf regression is caused by *not* inlining the aoti_torch_check. ```C10_UNLIKELY_OR_CONST``` is the cause for the slow compilation that #119220 tried to address. So, simply removing ```C10_UNLIKELY_OR_CONST``` can fix the slow compilation issue as well. Moreover, removing ```C10_UNLIKELY_OR_CONST``` allows us to inline the error check code and hence fixes the CPU perf regression, too.

Reply:
Got it.